### PR TITLE
Fixes issue #723 (and #645)

### PR DESCRIPTION
**Pull request by hannaei**
Defined a print method for SimulationBatch object

### CODE REVIEW

File: R/simulation-batch.R [hunk 1]
```diff
@@ -71,6 +71,25 @@ SimulationBatch <- R6::R6Class(
       batchRunValues <- SimulationBatchRunValues$new(parameterValues, initialValues)
       rClr::clrCall(self$ref, "AddSimulationBatchRunValues", batchRunValues$ref)
       return(batchRunValues$id)
+    },
+
+    #' @description
+    #' Print the object to the console
+    #' @param ... Rest arguments.
```
Comment (IndrajeetPatil) [inline R/simulation-batch.R:78]:
```suggestion
    #' @param ... Rest of the arguments.
```

Comment (msevestre) [reply R/simulation-batch.R:78]: isn't it called in fact Rest argument or (Rest parameter)? Or is it just in Javascript that it is called like this? Just curious

Comment (IndrajeetPatil) [reply R/simulation-batch.R:78]: Not in R, no. 

Typically, it is documented as 

- for `S3` methods (e.g., see `?base::mean`, `?tidyr::extract`, etc.)

```r
... further arguments passed to or from other methods
```

- for regular functions

```r
... Additional arguments
```

No hard and fast rules here, of course. 

So, if `Rest arguments` is the convention you have adopted, ignore my suggestion, but it's definitely not idiomatic of R package docs.

File: R/simulation-batch.R [hunk 2]
```diff
@@ -71,6 +71,25 @@ SimulationBatch <- R6::R6Class(
...
       return(batchRunValues$id)
+    },
+
+    #' @description
+    #' Print the object to the console
+    #' @param ... Additional arguments.
+    print = function(...) {
+      simulationBatchOptions <- rClr::clrGet(self$ref, "SimulationBatchOptions")
+      private$printClass()
+      writeLines(paste0("   ", capture.output(self$simulation)))
```
Comment (msevestre) [inline R/simulation-batch.R:82]: What does `capture.output` do?

Comment (hannaei) [reply R/simulation-batch.R:82]: It sends output to a character string.

Comment (msevestre) [reply R/simulation-batch.R:82]: What does that mean ? The result of print is concatenated to a string? Wouldn't printing the name of the simulation be enough?

Comment (hannaei) [reply R/simulation-batch.R:82]: Yes, it looks like this:
```
SimulationBatch: 
   Simulation: 
      Name: Simple 
      Source file: D:/github/OSPSuite-R/tests/data/simple.pkml 
   runValuesIds: 
   Parameters: Organism|Liver|Volume R1|k1 
   Molecules: Organism|Liver|A 
```
But we could only have the name printed, sure.

Comment (msevestre) [reply R/simulation-batch.R:82]: yeah. Let's remove the file name.  We do not need this I would think. Plus if we ever add more info in the simulation print., this is going to pollute this print

**Comment by msevestre** [comment]
#645 is probably not fixed automatically because the message just says and #645 instead of and fixes #645

**Pull request merged**